airbyte-integrations/connectors/destination-oracle/metadata.yaml

```diff
@@ -2,16 +2,12 @@ data:
   connectorSubtype: database
   connectorType: destination
   definitionId: 3986776d-2319-4de9-8af8-db14c0996e72
-  dockerImageTag: 0.2.0
+  dockerImageTag: 1.0.0
```

Comment:
do we really think this release is worthy of a 1.0.0. label?

Reply:
same comment as above